docs/cloud/how-to-get-started-with-temporal_cloud.md

```diff
@@ -0,0 +1,99 @@
+---
+id: how-to-get-started-with-temporal-cloud
+title: How to get started with Temporal Cloud
+sidebar_label: Get started
+description: Create an account, get certificates, create a Namespace, and invite users.
+tags:
+  - how-to
+---
+
+<!--- Onboarding guide for Temporal Cloud --->
+
+To get started with Temporal Cloud, following these onboarding steps:
+
+1. [Create an account.](#create-an-account-in-temporal-cloud)
+1. [Get CA certificates.](#get-ca-certificates)
+1. [Create a Namespace.](#create-a-namespace)
+1. [Invite users.](#invite-users)
+
+## 1. Create an account in Temporal Cloud
+
+:::caution
+
+The email address you use to create an account becomes the first global admin and can assign access to all Namespaces.
+Choose wisely.
```

Comment:
This is so strongly worded, we should specify what the implications are..can I change the email later? Can I assign other global admins? Is there something immutable I should know about?

Reply:
Good point. Unless I'm mistaken, membership in the Global Admin Role for a Namespace can be modified post facto. I'll update accordingly.